composer.json

```diff
@@ -19,6 +19,7 @@
     ],
     "require": {
         "php": "^7.2|^8.0",
+        "ext-bcmath": "*",
```

Comment:
Can this line be reverted? People who don't have this extension can still use moneyphp/money v3.

Reply:
No, this is required by moneyphp now and we also added this to Cashier Stripe. I don't expect this to have a huge impact as most PHP installs have this extension.